wenet/cif/attention.py

```diff
@@ -0,0 +1,291 @@
+# Copyright (c) 2023 ASLP@NWPU (authors: He Wang, Fan Yu)
```

Comment:
same as embedding.py, attention shares almost the same code as transformer/attention.py, and transformer/attention.py takes streamming into account.